_src/static/sass/components/_headings.scss

```diff
@@ -0,0 +1,128 @@
+// ------------------------------
+// edX Pattern Library: Components - Headings
+
+// About: Contains base styling for headings, relying on Bourbon's mathematical scales for precision.
+
+// Scale:
+// 12, 14, 16, 18, 21, 24, 32, 43
+// ----------------------------
+
+// #GLOBALS
+// #HEADINGS
+// H1
+// H2
+// H3
+// H4
+// H5
+// H6
+// ----------------------------
+
+
+// ----------------------------
+// #GLOBALS
+// ----------------------------
+.hd-1,
+.hd-2,
+.hd-3,
+.hd-4,
+.hd-5,
+.hd-6 {
+    font-family: $headings-base-font-family;
+    font-weight: $headings-base-font-weight;
+    color: $headings-base-color;
+
+    &.emphasized,
```

Comment:
Also, revisiting this, is this class/rule needed? I thought we were creating a base set of headings and then just allowing a a de-emphasized version of those (i.e. no emphasized third version). If this is the case, please remove.